doc: README bootloader update (#8831)
nrfconnect/sdk-nrf: doc/mcuboot/readme-ncs.rst

* `Customizing the bootloader`_
* `Firmware updates`_

When you add MCUboot to your application build, the files that can be used for firmware over-the-air (FOTA) upgrades are automatically generated.

> Read the `MCUboot output build files`_ page for the list of all the FOTA upgrade files that are automatically generated.

Reviewer: ```suggestion
See the `MCUboot output build files`_ page for a list of these files.
```

Let's use the same phrasing as below and avoid repeating ourselves.
